ext/net/ops.rs

```diff
@@ -740,6 +746,38 @@ fn rdata_to_return_record(
         .as_aname()
         .map(ToString::to_string)
         .map(DnsReturnRecord::Aname),
+      CAA => r.as_caa().map(|caa| {
+        let value = caa.value();
+
+        DnsReturnRecord::Caa {
+          critical: caa.issuer_critical(),
+          tag: caa.tag().to_string(),
+          value: match value {
+            Value::Issuer(name, key_values) => {
+              let mut s = String::new();
```

Comment:
TODO: utilise `name.to_string()` and `key_values.to_string()` instead of handrolling.

Unfortunately can’t just use `value.to_string()` as `trust-dns` wraps in unwanted additional double quotes - https://github.com/bluejekyll/trust-dns/blob/95b2dee327ade007bf317ca98bbd3b24c0bdd096/crates/proto/src/rr/rdata/caa.rs#L842